File: python/prism_sdk/throughput_capability_negotiation_integrity_inference.py
"""Backends P32 throughput inference capability-negotiation integrity feature."""
from .capability_negotiation_integrity_support import BackendCard7,BackendRequest4,CapabilityNegotiationIntegrityError,manifest,negotiate
FEATURE_ID="AFA-backends-P32-F03";CONTRACT_VERSION="backends-throughput_capability_negotiation_integrity_inference/1.0"
def throughput_capability_negotiation_integrity_inference_manifest():return manifest(feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="throughput",mode="inference")
def negotiate_throughput_capability_negotiation_integrity_inference(request:BackendRequest4)->BackendCard7:return negotiate(request,feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="throughput",mode="inference")
__all__=["FEATURE_ID","CONTRACT_VERSION","throughput_capability_negotiation_integrity_inference_manifest","negotiate_throughput_capability_negotiation_integrity_inference"]

File: crates/backends/src/throughput_capability_negotiation_integrity_inference.rs
//! Backends P32 throughput inference capability-negotiation integrity feature.
use super::capability_negotiation_integrity_support::{manifest,negotiate,BackendCard7,BackendRequest4,CapabilityNegotiationIntegrityError};
pub const FEATURE_ID:&str="AFA-backends-P32-F03";pub const CONTRACT_VERSION:&str="backends-throughput_capability_negotiation_integrity_inference/1.0";
pub fn throughput_capability_negotiation_integrity_inference_manifest()->serde_json::Value{manifest(FEATURE_ID,CONTRACT_VERSION,"throughput","inference")}
pub fn negotiate_throughput_capability_negotiation_integrity_inference(request:&BackendRequest4)->Result<BackendCard7,CapabilityNegotiationIntegrityError>{negotiate(request,FEATURE_ID,CONTRACT_VERSION,"throughput","inference")}
